### PR TITLE
Additional trending rewards work

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

Additional changes to finish trending page + playlist rending:
- Changes to tiles to show crowns
- Clickable rewards banner

### Dragons

None

### How Has This Been Tested?

Locally

### CODE REVIEW

File: src/components/track/mobile/PlaylistTile.module.css [hunk 1]
```diff
@@ -263,3 +263,7 @@
   opacity: 1;
   transition: opacity ease-in-out 0.5s;
 }
+
+.rankIcon {
+  margin: 0px 16px 0px 10px !important;
```
Comment (raymondjacobson) [inline src/components/track/mobile/PlaylistTile.module.css:268]: why'd this need !imp?

File: src/components/track/mobile/TrackTile.tsx [hunk 2]
```diff
@@ -63,6 +64,29 @@ const formatCoSign = ({
...
+  isVisible = true
+}: {
+  showCrown: boolean
+  index: number
+  isVisible?: boolean
+  className?: string
+}) => {
+  return (
+    <>
+      {isVisible && (
```
Comment (raymondjacobson) [inline src/components/track/mobile/TrackTile.tsx:80]: this is an interesting one to review for our little style guide. i know we had said x && <component>, but i think ternary with null might be more performant than this. no need for action on this PR, just somethin i thot of

File: src/containers/lineup/LineupProvider.tsx [hunk 3]
```diff
@@ -164,7 +164,7 @@ export interface LineupProviderProps {
 
   /**
    * Whether or not to apply leading element tile props and styles to the
-   * skeleton tile rendered in its place
+   * skeleton tile rendered in its placa
```
Comment (raymondjacobson) [inline src/containers/lineup/LineupProvider.tsx:167]: esqueleto en la placa

**Review by raymondjacobson — APPROVED** [review]

File: src/containers/lineup/LineupProvider.tsx (hunk 3 above)
Comment (piazzatron) [reply src/containers/lineup/LineupProvider.tsx:167]: lol

File: src/components/track/mobile/PlaylistTile.module.css (hunk 1 above)
Comment (piazzatron) [reply src/components/track/mobile/PlaylistTile.module.css:268]: I forget, some conflicting rule - will double check

File: src/components/track/mobile/TrackTile.tsx (hunk 2 above)
Comment (piazzatron) [reply src/components/track/mobile/TrackTile.tsx:80]: Interesting - why would ternary with null be more performant?

**Pull request merged**